### DISCUSSION TITLE
Remix Meetup Ideas

### DISCUSSION THREAD

**Discussion by markmals** [remix-austin/remixaustin-com#130] [Ideas]
I was talking with @sethdavis512 last month and came up with some ideas for future meetups that are not focused around talks. I wanted to share those here and start some discussion! (pun intended)

* Live coding with the "core" meetup members on some interesting Remix idea, with the opportunity for the audience to ask questions during the live coding session
* Hackathon where we can start at one meetup and finish at the next
    * Maybe we have coffee shop hangs midway through the month with no food or anything, but an opportunity to keep working on stuff and see each other ("Remix Austin Informal Hangout")
* Some collaboration with [Open Austin](https://www.open-austin.org/) and/or the [Austin Open Data Portal](https://data.austintexas.gov/)? Can we build something with Remix for the good of the city? Can we have people submit ideas for Austin Public Library website improvements?
* Let’s replace Meetup with a [federated](https://malstrom.me/blog/mastodon-guide) meeting platform built on Remix (pardon the self-plug)
* Rebuild the Remix Austin website from scratch
    * Start with a discussion post with a wireframe (I'm working on this, I swear!)
    * What other features can we have besides a landing page and a blog?

**Comment by grantnorwood** [comment]
Your hackathon idea sounds particularly neat!

And I'd love to hear everyone's ideas for anything website.  Might even be neat to imagine how we could integrate with Zoom _(e.g., publishing Zoom urls automatically, since Meetup doesn't let us do this sanely)_.

**Reply by markmals** [reply]
Yeah that's a great idea!

**Comment by brookslybrand** [comment]
Thanks @markmals, I think these are all great ideas and definitely want to empower y'all however possible!

The only thing I'm not super certain about is replacing Meetup. Personally I don't super love Meetup, so I'm not completely opposed. However, the one thing that makes it kind of hard is Meetup is the way we're able to make an [official network of Remix meetups](https://www.meetup.com/pro/remix-run/). Very open to exploring alternatives as an experiment with this meetup, but if we move to an alternative solution in my official role at Remix I'll have to figure out how to make it work for all meetups.

**Reply by grantnorwood** [reply]
Meetup.com is also a good place for folks to discover us, it feels like the go-to for anyone seeking out new groups.

**Reply by markmals** [reply]
Yeah I think the "replace Meetup" idea is more of a "far future" idea. Theoretically it would very much align with the idea of having a network of Remix meetups. Remix could host their own server and each meetup could have its own account on that server. Then people on other servers could see the events created by those official Remix accounts. But I do agree that it would make organic discovery more difficult.
